### ISSUE TITLE
Minor typo

### ISSUE THREAD

**Issue by snps-kyuhyun** [sj-li/DP_GAN#2]
Hello, I'm not sure this repository is under maintenance or not,

I report minor typo in `generator.py`.
In the line below, nn.BatchNorm2d(64) should be nn.BatchNorm2d(32).

https://github.com/sj-li/DP_GAN/blob/81ecf741764916a2cbbc7bfe0ddecaa5badd982c/models/generator.py#L28

Fixing this issue will help other users!
